### ISSUE TITLE
WE'RE HIRING.

### ISSUE THREAD

**Issue by Parkayun** [ironhee/ironhee-resume#1]
jobs@add2paper.com

**Comment by KimDarren** [comment]
@Parkayun Thanks to your awesome information. :) I will send my resume to the email right now.

**Comment by Parkayun** [comment]
@KimDarren We are hiring only frontend programmer.

**Comment by KimDarren** [comment]
@Parkayun Yes! I'm not only mobile developer but also fronted developer :)
